src/main/resources/db/migration/V70__add_indexes_for_child_tables.sql

```diff
@@ -0,0 +1,6 @@
+BEGIN;
+-------------------------------------------------------
+CREATE INDEX idx_identifier_type ON reference(identifier_type);
```

Comment:
The convention for our index names (which I think have to be unique in the database) is to use the table name

So these should `be idx_reference_identifer_type`